qubes/core2migration.py

```diff
@@ -75,11 +75,6 @@ def load_globals(self, element):
             self.default_netvm = int(default_netvm) \
                 if default_netvm != "None" else None
 
-        default_fw_netvm = element.get("default_fw_netvm")
-        if default_fw_netvm is not None:
-            self.default_fw_netvm = int(default_fw_netvm) \
-                if default_fw_netvm != "None" else None
-
```

Comment:
Why did this went away? This property is obviously present in whatever legacy xmls, shouldn't it be loaded and coerced like in `Qubes._migrate_however_it_was_called()`?

Reply:
In practice it wasn't used for anything but creating new ProxyVMs. You may think that it was value for `QubesProxyVM.netvm` if `QubesProxyVM.uses_default_netvm==True`, but the later was hardwired to `False`.